build/args/release.gn

```diff
@@ -1,7 +1,8 @@
 import("all.gn")
 is_component_build = false
-is_official_build = true
 is_component_ffmpeg = true
+is_official_build = true
+use_jumbo_build = true
```

Comment:
Can you move this to `all.gn`, we use jumbo in debug build as well.

Reply:
This is a great point. AFAIK there's no reason to not use jumbo build everywhere